src/home_work_05/task_03_get_ranges.py

```diff
@@ -0,0 +1,32 @@
+"""
+Реализовать функцию get_ranges
+которая получает на вход непустой список неповторяющихся целых чисел,
+отсортированных по возрастанию, которая этот список “сворачивает”
+get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+get_ranges([4, 7, 10]) // "4,7,10"
+get_ranges([2, 3, 8, 9]) // "2-3,8-9"
+"""
+
+
+def get_ranges(int_list):
+    range_list = []
+    sub_list = []
+    for elem in int_list:
+        if elem - 1 not in int_list or elem + 1 not in int_list:
```

Comment:
Parentheses wouldn't be redundant here ))

Reply:
solved as O(n) with simple if condition